kolibri/plugins/management/assets/src/vue/manage-content-page/wizard-export.vue

```diff
@@ -148,11 +204,21 @@
     margin-right: 0.2em
     margin-bottom: -6px
 
-  .drive-list
-    margin: 2em
-
   .drive-names
-    margin: 0.6em 0
+    padding: 0.6em
+    border: 1px $core-bg-canvas solid
```

Comment:
Any opinions on making the whole block clickable? I've got no issues with removing the border, just throwing out options. 

My original motivation for including borders was to clearly separate the drive and their attributes... but there are less properties available than I thought, so I can see an argument for getting rid of the borders.

@jtamiace?